pico_menu_rotary_test.py

```diff
@@ -50,25 +58,33 @@
     r=SCR_ROT)
 
 display.erase()
-display.fill_rectangle(0, 0, 240, 320, color=color565(0,0,0))    
-
-def menu_item(c1, c2, c3, x, y, font, Text):
-    display.set_pos(x, y)
+display.fill_rectangle(0, 0, 240, 320, color=color565(0,0,0))
+
+def menu_item(color, pos, font, Text):
+    """Menu item.
+
+    Args:
+        color: Color as a tuple (c1, c2, c3).
+        pos: Position as a tuple (x, y).
+        font: The font.
+        Text: The text to display.
```

Comment:
This docstring uses Google-style. See [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for more.